src/pydvl/value/shapley/types.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+from typing import Protocol
+
+import numpy as np
+from numpy._typing import NDArray
+
+
+class ShapleyMode(str, Enum):
```

Comment:
Moving this here makes it a bit more cumbersome to import and use for the users.

Reply:
I think I moved it to avoid some circular import. I'll try to move it back in the next PR.